lua/lspconfig/erlangls.lua

```diff
@@ -0,0 +1,31 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+
+configs.erlangls = {
+  default_config = {
+    cmd = { "erlang_ls" };
+    filetypes = {"erlang"};
+    root_dir = function(fname)
+      return util.root_pattern("rebar.config", "erlang.mk", ".git")(fname) or vim.loop.os_homedir()
```

Comment:
why homedir?

Reply:
because of https://github.com/neovim/nvim-lspconfig/blob/80290ccc3a624b515541d1bc4e26257abf4951de/lua/lspconfig/elixirls.lua#L9

Reply:
> because of
> 
> https://github.com/neovim/nvim-lspconfig/blob/80290ccc3a624b515541d1bc4e26257abf4951de/lua/lspconfig/elixirls.lua#L9

I suspect it's incorrectly using `os_homedir`. It's basically never what you want and it's often preferred to use `util.path.dirname(fname)`.

Reply:
So what would be a better fallback in this case?

Reply:
```lua
return util.root_pattern("rebar.config", "erlang.mk", ".git")(fname) or
  util.path.dirname(fname)
```

Reply:
thank you